requirements.txt

```diff
@@ -1,6 +0,0 @@
-
-radical.utils>=1.12
-radical.pilot>=1.18
-packaging
-requests>=2.4.0
```

Comment:
requirements files got deleted - that was accidentally, right?

Reply:
Thanks, resolved.